time_manager/time_manager.F90

```diff
@@ -1969,7 +2052,13 @@ function set_date_private(year, month, day, hour, minute, second, tick, Time_out
  case(THIRTY_DAY_MONTHS)
    set_date_private = set_date_thirty   (year, month, day, hour, minute, second, tick, Time_out, err_msg)
  case(GREGORIAN)
-   set_date_private = set_date_gregorian(year, month, day, hour, minute, second, tick, Time_out, err_msg)
+   old_method_local = .false.
+   if( present(old_method) ) old_method_local=old_method
```

Comment:
You need to explain the difference between the old and new methods in a comment

Reply:
Added an explanation in the description for the variable old_method